test/replication/prune.result

```diff
@@ -137,37 +137,28 @@ test_run:cmd('stop server replica1')
 ---
```

Comment:
If your test covers this old one completely, we could delete it, I think. Instead of fixing.

Reply:
Yeah, let's delete it.